pkg/tfbridge/provider.go

```diff
@@ -1230,6 +1228,9 @@ func (p *Provider) Update(ctx context.Context, req *pulumirpc.UpdateRequest) (*p
 	ic := newIgnoreChanges(ctx, schema, fields, olds, news, req.GetIgnoreChanges())
 	diff, err := p.tf.Diff(ctx, res.TFName, state, config, shim.DiffOptions{
 		IgnoreChanges: ic,
+		TimeoutOptions: shim.TimeoutOptions{
```

Comment:
It strikes me odd that ResourceTimeout is not set. Does this mean we ignore schema-derived timeouts during Update? That may be a bug..

Reply:
Could something depend on the previous behaviour? Are we aware of any especially slow diffing/destroying resources?

Reply:
This PR does not change behavior I hope but there's some fuzziness here so I will backlog in https://github.com/pulumi/pulumi-terraform-bridge/issues/1651 to investigate further .